alcs-api/src/main.ts

```diff
@@ -21,14 +22,28 @@ async function bootstrap() {
   const port: number = config.get<number>('PORT');
 
   // cors
-  app.enableCors();
+  app.enableCors({
+    origin: [
+      config.get<string>('BASE_URL'),
+      config.get<string>('KEYCLOAK.AUTH_SERVER'),
+    ],
+  });
 
   // swagger
   const documentBuilderConfig = new DocumentBuilder()
```

Comment:
nitpick: Should we move the swagger setup to its own enableSwagger() function? Then when we add a DEVELOPMENT flag we can wrap it in an if?